package.json

```diff
@@ -50,19 +50,22 @@
   },
   "scripts": {
     "build": "bun run clean && bun run scripts/build.ts",
-    "build:vsix": "bun run build --production",
     "clean": "git clean -xdf dist",
+    "generate": "fantasticon",
     "prepare": "husky"
   },
   "devDependencies": {
     "@commitlint/cli": "^19",
     "@commitlint/config-conventional": "^19",
     "@tsconfig/bun": "^1.0.7",
     "@tsconfig/strictest": "^2.0.5",
+    "@types/brace-expansion": "^1.1.2",
     "@types/bun": "^1.1.6",
     "@types/vscode": "^1.87",
     "@vscode/vsce": "^2.25.0",
+    "brace-expansion": "^4.0.0",
```

Comment:
eventually, this will need to be bundled with the extension, but I ran into an error when moving this to `"dependencies"` rather than `"devDependencies"`. need to look into it

Reply:
can address this as needed, later